---
title: Chemistry Review
permalink: /general-chemistry/
---

# Chemistry Textbook
* [Principles of Chemistry: The Molecular Science](https://www.amazon.com/gp/offer-listing/0495390798/) published by Cengage, 2010.
* Authors: [John W. Moore](https://www2.chem.wisc.edu/users/jwmoore), [Conrad Stanitski](https://www.fandm.edu/conrad-stanitski), [Peter C. Jurs](https://news.psu.edu/story/169275/2010/03/17/academics/peter-jurs-named-fellow-american-chemical-society) abbreviated as **MSJ**

## Chapter 5
* p. 143-145 Intro to acids (contributes H<sup>+</sup>) and bases (contributes OH<sup>-</sup>)
* p. 146 -148 Neutralization reactions. Acid + Base --> H<sub>2</sub>0 + Salt
* p. 150 Gas-forming exchange reactions, e.g.:
	* calcium carbonate wtih hydrochloric acid yields gaseous CO<sub>2</sub> and aqueous calcium chloride *aka*
	* CaCO<sub>3</sub> + 2 HCl (aq) --> CaCl<sub>2</sub> (aq) + *intermediate* **H<sub>2</sub>CO<sub>3</sub>** --> CaCl<sub>2</sub> (aq) + **H<sub>2</sub>O (liquid) + CO<sub>2</sub> (gaseous)**

### Redox Reactions, p. 151-158 
* See JH handwritten notes 8/06/2020 for redox diagram which draws on material from: Alberts p.71, Sadava p.167, MSJ p. 151-155, especially the table on MSJ p. 155.
* p. 152-154, common oxidizing and reducing agents 
* **LEO** = Oxidation causes Loss of Electrons *aka* **combines with oxygen**
* **GER** = Reduction causes Gain of Electrons *aka* loss of oxygen from the molecule *aka* **reduces the postive charge**.
	* eg, see what happens to Hydrogen in magnesium reaction on p. 646. The "positive **+1** charge" charge of aqueous HCl *reduces* its positivity to **0** in the resulting H<sub>2</sub> gas.

## Chapter 7: Quantum Theory, Structure of Atom, Electron Configurations, Atomic Orbitals, and Consequences for Periodic Table Trends

### The Four Quantum Numbers: *n*, &#8467;,  *m<sub>&#8467;</sub>*, *m<sub>s</sub>* – **MSJ** Chapter 7.5, p. 237-242

* List of four quantum numbers is [here](https://en.wikipedia.org/wiki/Quantum_number#Electron_in_an_atom) and a general description of atomic orbitals is on [this wiki page](https://en.wikipedia.org/wiki/Atomic_orbital#Types_of_orbitals).

#### Principal quantum number "*n*"
* Values = {1, 2, 3, 4,...} *aka* which electron shell, principal energy level, or "size" of the electron orbital. See also [the wiki article](https://en.wikipedia.org/wiki/Principal_quantum_number).
* *n=1* has 1 subshell s with 2 electrons. 
* *n=2* has 2 subshells. *2s<sup>2</sup> + 2p<sup>6</sup>* = total of **8 electrons** in this shell.
* *n=3* has 3 subshells. *3s<sup>2</sup> + 3p<sup>6</sup> + 3d<sup>10</sup>* = total of **18 electrons** in this shell.
* Etc.

#### Azimuthal quantum number "&#8467;" aka subshell
* Specifies the [quantum number for an atomic orbital](https://en.wikipedia.org/wiki/Azimuthal_quantum_number) which determines its orbital angular momentum as well as the **shape of the electron orbital**: s, p, d, f, g, etc...
* *&#8467;=0* = **subshell s**, maximum of 2 electrons. Spherical shape.
* *&#8467;=1* = **subshell p**, maximum of 6 electrons. Dumbell shapes, one for each axis: p<sub>x</sub>, p<sub>y</sub>, p<sub>z</sub>
* *&#8467;=2* = **subshell d**, maximum of 10 electrons. For shape, look at the third row of [this Wiki  image](https://en.wikipedia.org/wiki/File:Spherical_Harmonics.png)
* *&#8467;=3* = **subshell f**, maximum of 14 electrons. For shape, look at the fourth row of [this Wiki  image](https://en.wikipedia.org/wiki/File:Spherical_Harmonics.png)

#### Magnetic quantum number "*m<sub>&#8467;</sub>*"
* Values of *m<sub>&#8467;;</sub>* are dependent on the subshell / azimuthal quantum number *l* and describes the **orientation** of each atomic orbital "lobe". See sample values in the table in [this wiki article](https://en.wikipedia.org/wiki/Magnetic_quantum_number) or in MSJ p. 240, Table 7.3.
* For subshell s where *&#8467;=0*, there is only one value for *m<sub>&#8467;</sub>* which is *m<sub>&#8467;</sub>* = 0. Holds 2 electrons
* For subshell p where *&#8467;=1*, there are 3 possible values for *m<sub>&#8467;</sub>* which is *m<sub>&#8467;</sub>* = {-1,0,+1}. Holds 6 electrons.
* For subshell d where *&#8467;=2*, there are 5 possible values for *m<sub>&#8467;</sub>* which is *m<sub>&#8467;</sub>* = {-2,-1,0,+1,+1,+2}. Holds 10 electrons.
* For subshell f where *&#8467;=3*, there are 7 possible values for *m<sub>&#8467;</sub>* which is *m<sub>&#8467;</sub>* = {-3,-2,-1,0,+1,+1,+2,+3}. Holds 14 electrons.

#### Spin quantum number "*m<sub>s</sub>*"
* Describes the [spin of an electron](https://en.wikipedia.org/wiki/Spin_quantum_number), values of +1/2 or -1/2, and at most two electrons can occupy an atomic orbital. If they are paired, then the electrons must have opposite spin. [Wikipedia](https://en.wikipedia.org/wiki/Quantum_number#Electron_in_an_atom) uses the variable name *s* for spin quantum number but textbooks and [Khan Academy](https://www.khanacademy.org/science/physics/quantum-physics/quantum-numbers-and-orbitals/v/quantum-numbers) use the m<sub>s</sub> notation instead.

### Periodic Table Trends
* Chapter 7, sections 7.9 - 7.12, p. 254 - 269
* Examples of trends across periodic table: Isoelectricity / ion electron configs, size of atoms, size of associated ions, ionization energies, and electron affinities.

#### Isoelectronic Ions and Atoms
* **MSJ** Chapter 7.8 Ion Electron Configurations, p. 251-252
* **Isoelectronic** atoms and ions have the same number of electrons and in the same orbital configurations.
* Group 1-3 metals do this by becoming positive cations (aka lose electrons).
* Group 14-17 (aka 4A - 7A) nonmetals do this by become negative anions (aka gain electrons). They ultimately have the "desired" octet of valence electrons equivalent to the noble gas (Group 18) in  their associated period. For Group 14-17 ions, they stay in the same period. 
* For Group 1-3 metals, these atoms become cations by losing electrons to get "demoted" back to the noble gas in the previous period. 
See illustration on p. 251; all the species in the same row have the same color and are *isoelectronic* with each other.

#### Electron Affinity 
* **MSJ** Chapter 7.12, p. 260 - 261.
* Not to be confused with electronegativity, even though they seem similar in concept and share the same trends on the periodic table (aka highest in upper right near Fluorine, lowest near lower left near Francium).
* Electronegativity is a dimensionless measure whose range was arbitrarily set by Linus Pauling and only relevant within a molecule.
* In contrast, **electron affinity** is a directly measured *unit of energy* released when an electron is added to a neutral atom in the gaseous state to form a negative ion. aka, the amt of energy you need to **add* to a anion to get it to give up its extra electron(s) to become a neutral atom. 
* Thus, main group metals in Groups 1 and 2 have low electron affinities because it's super easy for them to give up electrons b/c they prefer to be positive cations.
* High electron affinity atoms "want" electrons more, e.g., non metals like Flurorine in Group 17 (7A) with low periods like period 2 or 3. These elements strongly prefer to be negative anions and you need to add a lot of energy to it to have them give up their extra electrons to become neutral atoms. 
* Some good answers on [this message board page](https://lavelle.chem.ucla.edu/forum/viewtopic.php?t=35085).

## Chapter 8: Covalent Bonding, Intro to Lewis Structures, Hydrocarbons, Molecular Orbital Theory

* p. 297 - contrast between *atomic* orbital theory and *molecular* orbital theory in Section 8.10 to explain **paramagnetism** in O<sub>2</sub> implying unpaired electrons, even though naive Lewis structure predicts **diamagnetism**
* p. 277 saturated hydrocarbons like butane VS. 2methylpropane branched chain
* p. 277 cyclopropane, cyclobutane, cyclopentane, cyclohexane 
* p. 278 C=O is a carbonyl group.
* p. 278-279 Aldehydes are hydrocarbons that have a -CHO functional group aka an "aldehyde functional group" which necessarily includes a carbonyl. 
* Whereas **alkanes** ( p.277) are necessarily saturated (aka have maximum # of hydrogens possible aka no double bonds between carbons), **alkenes** (p. 280) are hydrocarbons that have at least 1 C=C double bond. 
* **Alkynes** are unsaturated hydrocarbons as well, but unlike alkenes, alkynes have triple bonds between carbon atoms. 

## Chapter 9: Molecular Structures, VSEPR, Orbital Hybridization, Noncovalent Interactions, and Intermolecular Forces
* **MSJ** p. 310, Figure 9.4 table listing names and shapes of common molecular structures around a central atom: linear, trigonal planar, trigonal bipyramidal, tetrahedral, octahedral. 
* p. 316, Figure 9.6 table illustrating more exotic shapes with 5 or 6 shared electron bonds and/or lone pairs surrounding a central atom: trigonal bipyramidal, seesaw, T-shaped, linear, octahedral, square pyramidal, square planar
### Hybridization of orbitals, sigma bonds, pi bonds, steric number
* **MSJ** Chapter 9
* Lone pair = pair of valence electrons that are not bonded/shared covalently with another atom. Eg., ammonia NH<sub>3</sub> has steric number of 4 = 3 sigma bonds to hydrogen + 1 lone pair.
* Valence Shell Electron-Pair Repulsion (VSEPR) theory. See [Wiki article](https://en.wikipedia.org/wiki/VSEPR_theory) and MSJ Chapter 9.2 (p. 309-318)
* Interesting summary of sp linear, sp2 trigonal, sp3 tetrahedral at [this page](http://faculty.otterbein.edu/DJohnston/chem220/tutorial1/carbon.html)
* See also these videos at Khan Academy:
	* [*sp<sup>3</sup>* hybridized orbitals and sigma bonds](https://youtu.be/u1eGSL6J6Fo) by Sal
	* [Pi bonds and *sp<sup>2</sup>* hybridized orbitals](https://youtu.be/lJX8DxoPRfk) by Sal
	* [*sp<sup>3</sup>* hybridization](https://youtu.be/JHgTNNX01r4) by AAMC + Khan Academy
	* [Steric number analysis](https://youtu.be/BM-My1AheLw) by AAMC + Khan Academy
	* [*sp<sup>2</sup>* hybridization](https://youtu.be/ROzkyTgscGg) by AAMC + Khan Academy
	* [*sp<sup>1</sup>* hybridization](https://youtu.be/BRHfy7envyQ) by AAMC + Khan Academy
	* Not yet watched: [How to find the hybridization of atoms in organic molecules](https://youtu.be/GuaozMpFS3w) by AAMC + Khan Academy

### Section 9.5: Molecular Polarity
* p. 325 caused by both difference in electronegativity between different atoms in a molecule *and* the organization of charges in the molecular structure
* For example, compare CF<sub>4</sub> is nonpolar because of molecular symmetry while CHF<sub>3</sub> is *polar* because the hydrogen atom throws off tetrahedral symmetry - p. 327

### Section 9.6: Noncovalent Interactions
 * Noncovalent interactions refer to all forces of attraction apart from covalent bonds, ionic bonds, and metallic bonds. p. 329
 * Noncovalent bonds can refer to both intramolecular and intermolecular forces
 * Primary categories: London Forces, Dipole-Dipole Forces, and Hydrogen Bonds


## Chapter 12: Chemical Kinetics, Rate Law, Mechanisms, Catalysts, Enzymes
* Example of methane combustion in bunsen burner with and without fire. CH<sub>4</sub> + 2 O<sub>2</sub> --> CO<sub>2</sub> + 2 H<sub>2</sub>O - p. 415
* Four factors that impact speed of reaction when all reactants and products are in the same phase (eg all are gases or all are in solution) *aka* in a **homogeneous reaction**:
	1. molecular structure and bonds
	1. concentration
	1. temperature
	1. presence and concentration of catalyst (if any)
* In addition, if we are talking about input/output that are in different phases aka a **heterogeneous reaction**:
	1. area of reaction surface
	1. nature of reaction surface

### Example of crystal violet Cv<sup>+</sup> reaction rate (p. 417)
* Colorful crystal violet dye (abbreviated Cv<sup>+</sup>) combining with hydroxide ion (OH<sup>-</sup>) to produce colorless product CvOH\ on p. 417-419. The full formula is crystal violet chloride combining with sodium hydroxide to form the CvOH and sodium chloride:
	* **CvCl** (ions in solution) + **NaOH** (ions in solution) --> **CvOH** (colorless product) + **NaCl** (ions in solution)
* The notation for "concentration of" uses brackets. e.g., [Cv<sup>+</sup>] is pronounced "concentration of crystal violet dye".
* Average reainction rate is delta [Cv<sup>+</sup>] / delta time. And since the concentration of Cv<sup>+</sup> decreases over time but reaction rate is usually measured as a positive rate, the final equation has a negative in it. Average reaction rate from time t<sub>1</sub> to t<sub>2</sub> =   **- &#916;[Cv<sup>+</sup>] / &#916;t**

### Example of NO<sub>2</sub> --> NO + O<sub>2</sub>, (p. 419)
* The crystal violet Cv<sup>+</sup> example from p. 417, one mole of each reactant (aqueous Cv<sup>+</sup> and aqueous OH<sup>-</sup>) produces one mole of the colorless product CvOH.  A nice one-to-one ratio so the reaction rate is *the same* regardless of whether we are measuring **[Cv<sup>+</sup> cation in solution]**, **[OH<sup>-</sup> anion in solution]**, or **[CvOH molecule]**
* In contrast, consider the reaction **2 NO<sub>2</sub> (gas) --> 2 NO (g) + O<sub>2</sub> (g)**. There is a 2:1 ratio between the reactant NO<sub>2</sub> and the product O<sub>2</sub>. This means the reaction rate seems different depending on if we measure [NO<sub>2</sub>] or [O<sub>2</sub>].
* To resolve this, we define reaction rate using stoichiometry with Equation 12.3 (p. 419): 
	* Let lowercase a,b,c,d be coefficients and upper case A,B,C,D be chemicals in the following reaction formula: aA +bB --> cC + dD
	* Then the uniform reaction rate is equivalent for the following:
		* Rate = - (1/**a**)(&#916;[**A**]/&#916;t) *reactant A*
		* Rate = - (1/**b**)(&#916;[**B**]/&#916;t) *reactant B*
		* Rate = + (1/**c**)(&#916;[**C**]/&#916;t) *product C*
		* Rate = + (1/**d**)(&#916;[**D**]/&#916;t) *product D*


### Sections 12.2 - 12.4: Rate Laws for 0th-, 1st-, and 2nd-order Reactions; Unimolecular, Bimolecular, and Termolecular Elementary Reactions; E<sub>a</sub>, exothermic and endothermic rxns(p. 422-436)
* Constant k is the specific rate a reacion will run at a given temperature, multiplied by a relevant concentration of a reactant or product.
	* **Zeroth Order Reaction = k * [A]<sup>0</sup>** = k * 1 = k. means that reaction proceeds independently of concentration of A.
	* **First Order Reaction = k * [A]<sup>1</sup>** reaction directly proportional to concentration of A
	* **Second Order Reaction = k * [A]<sup>2</sup>** means reaction proportional to square of [A]
	* For more info see [this page](https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Kinetics/Rate_Laws/The_Rate_Law)
* All reactions can be decomposed to **elementary reactions** (aka subreactions). which are themselves **unimolecular**, **bimolecular**, or **termolucular** (requiring 3 reactants). 
	* Per [Wikipedia](https://en.wikipedia.org/wiki/Molecularity#Termolecular_reactions), "Reactions of higher molecularity [than 3] are *not observed* due to very small probability of simultaneous interaction between 4 or more molecule." 
	* See [this video](https://www.youtube.com/watch?v=S84Llf1vqiM) for a good explanation of how to express rate laws for elementary reactions involving 1, 2, or 3 reactants.
* **Activation Energy** (abbreviated **E<sub>a</sub>**) needed to get over the hump plus if the end state is lower energy, the reaction is *exothermic*. Conversely if final product is higher energy state, then the reaction is *endothermic*. (p, 434)
* Intermediate form in the process of moving from reactant to product is called the **transition state** aka **activated complex**. (p. 433)
* See example of transition between *cis*-2-butene to *trans*-2-butene on p. 433-434.

### Section 12.5: Arrhenius Equation to determine k
* k = Ae <sup>-E<sub>a</sub>/RT</sup> where:
	* **k** = rate constant
	* **A** = Arrhenius frequency factor, which depends on (a) how often molecules collide when all concentratrions are at 1 mol per liter and on (b) whether molecules are properly oriented when they collide
	* **E<sub>a</sub>** = activation energy
	* **R** = [gas law constant](https://en.wikipedia.org/wiki/Gas_constant), 8.314 joules/ (mol * temperature in degrees Kelvin)
	* **T** = temperature in degrees Kelvin

## Chapter 13: Chemical Equilibrium
## Chapters 14-15: Acids, Bases, Buffers


# Biology textbook
* *Life, The Science of Biology*, [Tenth Edition](amazon.com/Life-Science-Biology-Sadava-Hardcover-dp-B00MEYGNSS/dp/B00MEYGNSS) published 2014. Abbreviated as **Sadava**.
* Authors: [David Sadava](https://www.cmc.edu/news/professor-and-biologist-david-sadava-to-retire), [David M. Hillis](https://sites.cns.utexas.edu/hillislab/lab-members), [H. Craig Heller](https://profiles.stanford.edu/h-craig-heller), and [May R. Berenbaum](https://en.wikipedia.org/wiki/May_Berenbaum)

## Chapter 2: Review of [General Chemistry](/general-chemistry/)

### Section 2.1: How does Atomic Structure Explain the Properties of Matter?
* Review of [atomic orbitals](https://en.wikipedia.org/wiki/Electron_shell) p. 24-25 
	* An atomic orbital can hold at most two electrons.
	* Remember:
		* First Shell: *1s<sup>2</sup>* = total of **2 electrons** in this shell. Only an s-orbital shape and 1 orbital.
		* Second Shell: *2s<sup>2</sup> + 2p<sup>6</sup>* = total of **8 electrons** in this shell. Two subshells. The *s* subshell has one spherical orbital in it. The *p* subshell has 3 dumbbell-shaped orbitals. Thus there are a total of 4 orbitals.
		* Third Shell: *3s<sup>2</sup> + 3p<sup>6</sup> + 3d<sup>10</sup>* = total of **18 electrons** in this shell. Three subshells: s, p, d. As expected, the s subshell contains one s orbital. The p subshell contains 3 p orbitals. The d subshell has 5 d orbitals. 1+3+5= **9 orbitals** which collectively hold a maximum of 18 electrons.
		* Fourth Shell: *4s<sup>2</sup> + 4p<sup>6</sup> + 4d<sup>10</sup> + 4f<sup>14</sup>* = total of **32 electrons** in this shell. 1+3+5+7 = **16 orbitals** which can hold a maximum of 32 electrons  
		* Fifth Shell: *5s<sup>2</sup> + 5p<sup>6</sup> + 5d<sup>10</sup> + 5f<sup>14</sup> + 5g<sup>18</sup>* = total of **50 electrons** in this shell.
	* Filling order is based on [the Aufbau principle](https://en.wikipedia.org/wiki/Aufbau_principle) aka the [Madelung energy ordering rule](https://en.wikipedia.org/wiki/Aufbau_principle#Madelung_energy_ordering_rule):
		* 1s, 2s, 2p, 3s, 3p, 4s, 3d, 4p, 5s, 4d, 5p, 6s, 4f, 5d, 6p, 7s, 5f, 6d, 7p, 8s, etc.

### Table 2.1 Chemical Bonds - p. 26

| Type of Bond              | Description                                                         | Bond Energy (kcal/mol) |
|---------------------------|---------------------------------------------------------------------|------------------------|
| Covalent bond             | Sharing of electron pairs                                           | 50-110                 |
| Ionic bond          | Example NaCl — electrons stolen to complete octets in valence shell to produce *+cations* and *-anions*   | 3-7                    |
| Hydrogen bond             | Attraction between bonded H<sup>+</sup> atom and electronegative atom. Important for [secondary structure within a protein molecule](https://en.wikipedia.org/wiki/Protein_secondary_structure) but also occurs between molecules   | 3-7                    |
| Hydrophobic interaction   | Nonpolar substances in presence of water (or other polar substrate) | 1-2                    |
| van der Waals interaction | Interaction of electrons of nonpolar substances                     | 1                      |


### Concept of Electronegativity - p. 28
* When two atoms of different elements are covalently bonded in a molecule, their electrons are not equally shared. So one of the positively charged nuclei may exert a greater attractive force on the shared electron pair than the other nucleus. This attractive force is called **electronegativity**.
* p. 28: atoms with similar electronegativity = **nonpolar covalent bond**. Atoms with different electronegativity produce a **polar covalent bond**.
* Extreme electronegativity = powerful positive proton nuclues which leads to ability to attract greater than "fair share" of shared electrons in a molecule
* Maps to having almost having enough electrons in valencce shell to complete s2p6 octet
* In general, most electronegative columns in the periodic table are Groups 16 & 17 because they have 6 & 7 electrons, respectively in their outer valence shell.
* Conversely, the least electronegative groups in the periodic table are Groups 1 & 2 which respectively have 1 & 2 electrons in their outermost shell and these are the easiest electrons to lose.
* At the limit, when superstrong-electronegative and superweakly-electronegative elements interact, they don't just form a hyper-polar covalent bond; instead actually turn into an Ionic Bond.
* good illustration of this in **MSJ** p. 288, Figure 8.5.
### Ionic Bonds - p. 28-29
* When 2 interacting atoms have a large enough difference in electronegativity, ions form as the more powerfully electronegative nucleus strips electrons away from the weakly electronegative (aka positive atom). 
* Example of sodium Na and chlorine Cl
	* Na has its 1st two shells filled (aka *1s<sup>2</sup>* + *2s<sup>2</sup> + 2p<sup>6</sup>*). Also, as a [Group I element](https://iupac.org/what-we-do/periodic-table-of-elements/), it also has an extra electron in the next subshell: 3s<sup>2</sup> aka the first electron to appear in Shell M. It is easy to lose this electron.
	* Meanwhile, Cl only has 7 electrons in its  *3s<sup>2</sup> + 3p<sup>6</sup>* subshells. But ideally it would "like" **8 electrons** to complete subshell *3p<sup>6</sup>*. 
	* So in solution, Cl's more electronegative (aka more positive) nucleus "steals" the sole electron in Na's outermost subshell 3s<sup>2</sup>.
	* This produces a **positive cation** Na<sup>+</sup> and a **negative anion**  Cl<sup>-</sup> .
* This also explains the octet rule for molecules formed by atoms where *s<sup>2</sup> and p<sup>6</sup>* subshells are concerned.

### Hydrogen bonds
* Salts and interactions with polar molecules - p. 29
* Good interactive page on [hemoglobin and primary, secondary, tertiary, and quartenary structure](https://www.labxchange.org/library/items/lb:LabXchange:99c39c9f:lx_simulation:1)

### Section 2.4: H<sub>2</sub>0 has special properties - p. 32-34
* **Electronegativity causes strongly polar covalent bonds.** Water has a tetrahedral shape with oxygen in the center, two of the tetrahedral vertices occupied by hydrogen atoms, and the two remaining vertices occupied by an unbonded electron pair. Because oxygen has a high electronegativity relative to hydrogen, there is a  partial *delta-positive* charge on each hydrogen. Conversely, there is a partial *delta-negative* charge on each unbonded electron pair on the other two vertices. 
* Highly polar bonds then cause multiple and powerful hydrogen bonds between water molecules. Can also refer to these hydrogen bonds as intermolecular forces.
	* In a liquid state, water molecules are connected to each other by chains of powerful but intermittent hydrogen bonds. Special properties of liquid water thanks to multiple strings of hydrogen bonds:
		* High surface tension
		* High cohesion (water to water) and high adhesion (water to other substances) which are critical to [upward water flow in plant xylem](https://en.wikipedia.org/wiki/Xylem#Main_function_–_upwards_water_transport) 
		* High surface tension (water striders) and allows beads of water droplets (ants carrying a ball of water)
		* Lower vapor pressure (aka evaporation)
		* High specific heat
		* High boiling point aka heat of vaporization
	* When water freezes and turns into a solid, the daisy chain of randomly appearing/disappearing hydrogen bonds form into more permanent hydrogen bonds. These new hydrogen bonds follow from the partial delta-charges; the result is a  "hollow" crystal structure for ice which is why is less dense than liquid water. Each water molecule is connected to [4 other water molecules in a 3-d tetrahedral structure](https://www.youtube.com/watch?v=BxCnDJSE1ck&t=157s). 

## Chapter 3: Macromolecules I - Proteins, Carbohydrates, Lipids (p. 39-61)
* Four kinds of macromolecules: proteins, carbohydrates, lipids, and nucleic acids
* Macromolecular polymers are made of different types of monomers:
	* Proteins are made up of **amino acid** monomers.
	* Carbohydrates are made up of simple **sugar** monomers.
	* Lipids are made up of lipid monomers connected by *noncovalent bonds*
* In general, only polymers with molecular weights > 1000 are considered macromolecules.
* Figure 3.1 (p. 40) has a good diagram of various functional groups, categories of compounds, and example compounds.
* Figure 3.3 (p. 41) has good % breakdown of macromolecules in living tissue.
* Reactants Monomer<sub>1</sub> + ... + Monomer<sub>n</sub> --> Polymer via removal of H<sub>2</sub>0 and addition of energy (endothermic reaction). Also called **condensation** or **dehydration reaction**.
* Reverse reaction of Reactant Polymer ---> breaking into smaller monomers by adding water and releasing energy from the reactant  (exothermic reaction). Called **hydrolysis**.

### Proteins
* The protein alphabet of monomers cosists of 20 amino acids. All proteins consist of a permutation of these 20 elemental units. 
	* The smaller limit of proteins is around **51 amino acids** for *[insulin](https://en.wikipedia.org/wiki/Insulin)* with molecular weight of 5,733.
	* The upper limit of protein chain size is around **27,000 amino acid** for the muscle protein *[titin](https://en.wikipedia.org/wiki/Titin)* with a molecular weight of almost 3 million.
* Polypeptide chains are linear (*aka* unbranched) sequences of covalently linked amino acids.
* Amino acids have 3 functional groups plus 1 hydrogen surrounding the central *&#945;*-carbon:
	1. **carboxyl functional group** -COOH ionizes to -COO<sup>-</sup>
	1.  **amine group** -NH<sub>2</sub> ionizes to -NH<sub>3</sub><sup>+</sup>
	1. **R side chain**
* Amino acids can appear as optical isomers with **D- for righthand** and **L- for lefthand** configurations. (d-amino acids for the Latin *dextro*; l-amino acids for the Latin *levo*). Only L- configurations appear in nature / are used by living cells with the [rare exception of some structural proteins used in bacterial cell walls](http://www.phschool.com/science/biology_place/biocoach/bioprop/landd.html).
* See Table 3.2 on p. 44 for good categorization of 20 amino acids based on R side chains.
* Amino acids are combined using a condensation / dehydration reaction that forms a **peptide linkage**. The N- terminus all the way on the left of the protein is by considered the "beginning" of the sentence and the C- terminus all the way at the right end is considered the "end" of the protein "sentence".
* Primary structure is the sequence of repeating -N-C-C-N-C-C-
* Good diagram of primary, secondary, tertiary, and quartenary structure on p. 46.
* Each monomer is an amino acid. A short sequence of amino acids is called a **peptide**. Multiple peptides together form polypeptides and protein subunits.
* Concept of [molecular chaperones](https://en.wikipedia.org/wiki/Chaperone_(protein)) which help guide proper folding of proteins directly after peptide synthesis or after denaturation to refold. Cancer cells produce chaperones to help secure proper folding for cancer-promoting proteins (p. 51). See p. 51 also for example of HSP60 cage + lid.

#### Secondary structure of proteins
* *&#945;*-helices are right-handed and the R-groups extend outward from the central axis of the -N-C-C-N-C-C- chain in a steady spiral. Coiling results from the hydrogen bonding between the small &#948;<sup>+</sup> of the H<sup>+</sup> on the N and the &#948;<sup>-</sup> on the carbonyl oxygen C=O of another amino acid in the chain.
* Pleated  *&#946;*-sheets

#### Tertiary structure of proteins
* Definitive 3D structure of a protein, which usually includes a polar (hydrophilic) exterior region and nonpolar (hydrophobic) interior where reactions happen.
* &#945;-helices and  &#946; pleatedsheets secondary structures are determined by hydrogen bonds. Meanwhile, tertiary structure is caused by nonovalent interactions between the **various -R side chains functional groups** hanging off the central polypeptide -N-C-C-N-C-C- . Examples:
	* Covalent disulfide bridges
	* Hydrogen bonds between side chains
	* Hydrophobic side chains an aggregate together in the interior of the protein. These close interactions are stabilized by van der Waals forces.
	* Ionic bonds can form salt bridges between positively and negatively charged side chains.
* p. 48 interesting example of how [Christian Anfinsen used ribonuclease A](https://en.wikipedia.org/wiki/Anfinsen%27s_dogma) to prove that primary structure contains all the info needed to determine tertiary structure.

#### Quartenary structure
* When a protein consists of two or more polypeptides, called **subunits**, each of these subunits fold into their own tertiary structure.
* The interaction of these subunits is the **quartenary structure**.

#### Proteins and associated functions - p.42

| Category                 | Function                                                    |
|--------------------------|-------------------------------------------------------------|
| Enzymes                  | Catalyze biochemical reactions                              |
| Structural proteins      | Provide physical stability and movement                     |
| Defensive proteins       | Recognize and react to nonself substances (e.g. antibodies) |
| Signaling proteins       | Control physiological processes (e.g. hormones)             |
| Receptor proteins        | Receive and respond to chemical signals                     |
| Membrane transporters    | Regulate passage of substances across cell membranes        |
| Storage proteins         | Store amino acids for later use                             |
| Transport proteins       | Bind and carry substances within the organism               |
| Gene regulatory proteins | Determine rate of expression of genes                       |


### Carbohydrates - p. 51
* Usually have the generic formula C<sub>m</sub>H<sub>2n</sub>O<sub>n</sub> where m,n &#8712; {positive integers}.
* Carbohydrates have important roles in energy storage/transport and structure like cellulose in plants. 
* *Chemically modified carbohydrates* also play critical structural roles in biology:  
	* **galactosamine** is a major part of cartilage (p. 55)
	* a derivative of **glucosamine** is a component of chitin which is important in arthropods and fungi.
* Four categories of carbohydrates based on number of monomers:
	1. **Monosaccharides** like glucose
	1. **Disaccharides** like sucrose, a combination of glucose and fructose
	1. **Oligosaccharides** are made up of about 3-20 monosaccharides
	1. **Polysaccharides** are made up of hundreds or thousands of monosaccharides. Examples: starch, glycogen, cellulose.

#### Monosaccharides
* In living systems, most monosaccharides are the dextro- / D- / (right-handed) optical isomer.
* *Pentoses* are five-carbon sugars, including [ribose](https://en.wikipedia.org/wiki/Ribose) which is part of the sugar-phosphate backbone of RNA and [deoxyribose](https://en.wikipedia.org/wiki/Deoxyribose) which forms the sugar-phosphate backbane of DNA.  
* *Hexoses* are 6-carbon sugars such as glucose, fructose, mannose, and galactose.
	* **Glucose** are 6-carbon sugars that can take the form of straight carbon chains or rings in chair configuration. The ring consists of 5 carbons and an oxygen; the 6th carbon hangs off of the ring at the 5-carbon.
	* Glucose comes in [both &#945;- and &#946;-isomers](https://www.worldofmolecules.com/3D/what-is-the-difference-between-alpha-and-beta-glucose.html) which convert to each other readily in aqeous solution and are in equilibrium.
* Dehydration synthesis reactions connects monosaccharides into disaccharides, oligosaccharides, and polysaccharides via [glycosidic linkages](https://en.wikipedia.org/wiki/Glycosidic_bond)

#### Polysaccharides
* Whereas all proteins are linear chains of amino acids, monosaccharides can bond in many branching ways to form polysaccharides.
* **Starches** are polysaccharides which are often [branched](https://en.wikibooks.org/wiki/Structural_Biochemistry/Carbohydrates/Polysaccharides) made up of glucose monomers via with &#945;-glycosidic linkages. 
	* These branches attach from the via 1-&#945; carbon of glucose monomer A to either the 4 or 6 carbon on the glucose molecule B.
	* Starch is the primary energy storage for plants. Some plant starches are unbranched like [amylose](https://en.wikipedia.org/wiki/Amylose) which can consist of 300-3000 (or even more) monomeric glucoses.
* **Glycogen** is highly branched and used in animals. 
	* Similarly to starch for plants, glycogen acts as an energy store for animals in the liver and in the muscles. 
	* The branches occur every 10-14 glucose monomers between the carbon 6 of the main branch and the 1-&#945; carbon of the branched glucose. See this [Wiki link on the brancher enzyme](https://en.wikipedia.org/wiki/Glycogen_branching_enzyme#Function) for more.
	* Unlike glucose which is readily soluble in water, glycogen is nonpolar and thus does not dissolve in water which relieve osmotic pressure in animal cells allowing more compact storage. 
* **Cellulose** is a critical componet of plant cell walls. Connected via glycosidic linkages at the 1-&#946; carbon (instead of the 1-&#945; carbon seen in starch and glycogen). This makes it much harder to break down cellulose.
* Chemically modified carbs which have undergone redox reactions and/or had functional groups like phosphate, amino, or N-acteyl gropus added. Cartilage and chitin.
* Also refer to carbohydrate panels in Alberts 5th, p. 112-113.

### Lipids and phospholipids - p. 56
* Types of lipids
	* Fats and oils (triglycerides) store energy.
	* Phosopholipids provide structure to membranes.
	* Carotenoids and chlorophylls capture light energy for plants.
	* Steroids and modified fatty acids play regulatory roles as hormones as vitamins.
	* Fat in animals provides thermal insulation
	* Myelin (fatty sheath) surrounding neuronal axons provide electrical insulation to speed the action potential.
	* Oil and wax on the surface of skin and leaves helps repel water and prevents excess evaporation.
* Simple lipids like fats and oils consist of **triglycerides**. 
	* A triglyceride typically combines an alcohol (like glyceryl with 3 -OH hydroxyl groups) with three fatty acids which are simply long chains of hydrocarbons. 
	* The glycerol and fatty acids are bonded by 3 dehydration/condensation synthesis reactions that yield 3 water molecules.
	* The resulting bonds are ester linkages with a central oxygen between each fatty acid and the glycerol molecule.
	* Saturated fatty acid = full of hydrogen atoms, no double bonding
	* Unsaturated fatty acid = some missing hydrogen atoms *aka* some double or triple bonding somewhere.
* **Phospholipids** are amphipathic; have a polar phosphate end and a nonpolar hydrocarbon chain end.
* **Carotenoids** are a family of light-absorbing pigments found in plants and animals. &#946;-carotene aka betacarotene is part of the photosynthesis process for plants. Meanwhile, betacarotine breaks down to two vitamin A molecules for humans as part of the visual system.
* **Steroids** are multiply linked rings of hydrocarbons that are important as membrane components as well as signals in the hormone system.
* **Vitamins** A, D, E, and K are lipids, aka they are fat soluble.
* **Waxes** are hydrophobic and at room temperature, malleable/plastic. Each wax molecule sonists of a saturated long-chain fatty acid connected via an ester linkage to a long-chain alochol. As such, they have between 40-60 CH<sub>2</sub> groups.
* Lipid panels in **Alberts 5th Ed**, p. 114-115
	* Stearic acid (C18) and palmitic acid (c16) are saturated aka full of hydrogens and only carbon single bonds.
	* Oleic acid (C18) is unsaturated with a single double bond between C9 and C10.
	* Glycerol is a 3-carbon alcohol with an -OH hydroxyl attached to C1, C2, and C3 respectively. It serves as the backbone for triglycerides when 3 fatty acids attach via dehydration/condensation synthesis at the ester linkage.
	* Carboxyl groups -COOH give rise to: (a) ionized form -COO<sup>-</sup>, (b) esters, (c) amides.
	* Phospholipids where one of the 3 fatty acids attached to the glycerol backbone is replaced by a hydrophilic  **phosphate group -PO<sub>4</sub><sup>3-</sup>** . Thus forming a hydrophilic head and 2 hydrophobic tails. 
	* The phosophate group can also attach to other moeities, such as choline in the example of [phosphatidylcholine](https://en.wikipedia.org/wiki/Phosphatidylcholine).
	* Isoprene units like [2-methyl-1,3 butadiene](https://en.wikipedia.org/wiki/Isoprene). See also the [Organic Chemistry section](/organic-chemistry/).


## Chapter 4: Macromolecules II - Nucleic Acids (p. 62-76)
* See Alberts 5th edition 


## Chapter 12: Inheritance, Genes, Chromosomes (p. 232 - 258)
* p. 234 - Character is a phenotypic attribute and a trait is a particular value for that attribute.
* A gene can have more than 2 alleles in the population. Example of the C gene in rabbits that determines rabbit coat color (p. 242):
	* The dominant wild-type *C* allele produces a dark gray coat
	* *c<sup>chd</sup>* recessive allele produces a [Chinchilla coat](https://en.wikipedia.org/wiki/Chinchilla_rabbit)
	* *c<sup>h</sup>* recessive allele produces a Himalayan coat aka [point coloration](https://en.wikipedia.org/wiki/Point_coloration#In_rabbits_and_rodents) where color is restricted to ears, feet, and other tips.
	* *c* recessive allele produces an [albino rabbit](https://www.cuteness.com/article/albino-rabbits) with white fur
* asdkfhja

---

# Molecular Biology of the Cell
* Alberts et al., Fifth Edition (2008)
* p. 20 - defintion of **homologs**: All related genes that result from duplication 
	* Duplicate genes that are conserved in two branches after speciation are called **orthologs**
	* Duplicate genes within a single genome are **paralogs**
	* Orthologs and paralogs are subsets of the larger containing set of homologs
* Dehydration Synthesis (aka Condensation Synthesis) vs. Hydrolysis. See Alberts 5e Fig. 2-64 and 2-65 on p. 85, Alberts 5e Fig. 2-19 on p. 57, and JH handwritten notes (8/22/2020)

### 10 Steps of Glycolysis

Glycolysis converts glucose to pyruvate in 10 steps. For more details, see JH handwritten notes August 7-13, 2020,  Alberts p. 120-121 (Panel 2-8), and [this page](/glycolysis). 


## Chapter 3: Proteins
* See p. 127-129 to see tables and panels of categories of polar vs. nonpolar sidechain/amino acids, acidic vs. basic sidechains, etc.



